### ISSUE TITLE
DualShock 4 Is Not Properly Detected

### ISSUE THREAD

**Issue by Saroufim** [RobLoach/net.lutris.Lutris#30]
Through Lutris, installed via rpm on Fedora 30, DualShock 4 Slim Pro is properly detected in video games and if it is not supported by the game itself, it is recognized automatically as an Xbox One controller. 

Via this flatpak bundle, Lutris does not detect the DualShock 4 Slim Pro controller and instead it lists it as an unknown wireless controller. The buttons are not properly mapped and one of the axes seems to be constantly pinging, making the game unplayable with the controller active. 

Extra note: The RPM-based Lutris displays the same issue when wine is not installed on the host system. This issue is resolved automatically when a complete version of wine is installed.

**Comment by gasinvein** [comment]
So is this specific to wine? Does native games recognize the controller correctly?

**Comment by Saroufim** [comment]
Correct. It's wine-specific and all native games (including SDL ones) recognize the controller properly. Flatpaked Steam also recognizes it instantly.

**Comment by gasinvein** [comment]
Now libusb is build with udev. Did it help?

**Comment by Saroufim** [comment]
Unfortunately, no. Gamepads are still undetected. Wired or wireless.

**Comment by Saroufim** [comment]
Controller now works with Wine but it still not detected by libretro. The Retroarch flatpak detects it tho.
